libcudacxx/docs/extended_api/synchronization_primitives/barrier/barrier_arrive_tx.md

```diff
@@ -29,7 +29,7 @@ transaction count.
 ## Effects
 
 * This function constructs an arrival_token object associated with the phase
-  synchronization point for the current phase. Then, decrements the expected
+  synchronization point for the current phase. Then, decrements the
```

Comment:
The "expected arrival count" - the value the barrier is reset to after a phase change - is only decremented by `arrive_and_drop`, arrive decrements the arrival count (the count that is tested against zero, and on a phase change is reseted to the "expected arrival count").